.github/workflows/ci-build.yml

```diff
@@ -98,7 +89,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, macos-latest]
-        python-version: [3.6, 3.7, 3.9]
+        python-version: ["3.7", "3.8", "3.9", "3.10"]
```

Comment:
If the full test suite is running with Python 3.10 there is no need to re-run the basic test suite with this version.